components/shooter.py

```diff
@@ -115,3 +128,4 @@ def execute(self) -> None:
 
         flywheel_request = VelocityVoltage(self.desired_flywheel_speed)
         self.flywheel.set_control(flywheel_request)
+        self.flywheel1.set_control(flywheel_request)
```

Comment:
One of the motor controllers should be following the other. The follow mode is more efficient.